src/bot_quake.py

```diff
@@ -68,18 +67,22 @@ async def file_reader(update, context):
                 "Inserire un numero da 1 a 10 rilevati caratteri non numerici"
             )
             return
-            
-
-    filename = f"https://webservices.ingv.it/fdsnws/event/1/query?starttime={intervallo_date[0]}T00%3A00%3A00&endtime={intervallo_date[1]}T23%3A59%3A59&minmag=-1&maxmag={massima_magnitudo}&mindepth=-10&maxdepth=1000&minlat={zona.minlat}&maxlat={zona.maxlat}&minlon={zona.minlon}&maxlon={zona.maxlon}&minversion=100&orderby=time-asc&format=text&limit=100"
-
+    filename = (
+        f"https://webservices.ingv.it/fdsnws/event/1/query?starttime="
+        f"{intervallo_date[0]}T00%3A00%3A00"
+        f"&endtime={intervallo_date[1]}T23%3A59%3A59&minmag=-1&maxmag="
+        f"{massima_magnitudo}&mindepth=-10"
+        f"&maxdepth=1000&minlat={zona.minlat}&maxlat={zona.maxlat}"
+        f"&minlon={zona.minlon}&maxlon={zona.maxlon}"
+        f"&minversion=100&orderby=time-asc&format=text&limit=100"
+    )
```

Comment:
Che ve ne pare di definire una funzione separata che preso in input i parametri ritorna l'url già pronto da chiamare ?

Così da rendere più chiaro cosa si sta facendo in questa sezione del codice.

Reply:
Fatto!